crates/wasmtime/src/runtime.rs

```diff
@@ -701,6 +702,16 @@ impl Engine {
     pub fn config(&self) -> &Config {
         &self.config
     }
+
+    pub(crate) fn get_compiler(&self) -> Compiler {
```

Comment:
Could an instance of `Compiler` be stored within the `Engine` itself, so the same instance is reused across all compilations? I think `Compiler` should be `Send` and `Sync` now right?